homeassistant/components/upnp/binary_sensor.py

```diff
@@ -23,10 +30,14 @@ async def async_setup_entry(
 
     LOGGER.debug("Adding binary sensor")
 
-    sensors = [
-        UpnpStatusBinarySensor(coordinator),
-    ]
-    async_add_entities(sensors)
+    async_add_entities(
+        UpnpStatusBinarySensor(
+            coordinator=coordinator,
+            entity_description=entity_description,
+        )
+        for entity_description in BINARYSENSOR_ENTITY_DESCRIPTIONS
+        if coordinator.data.get(entity_description.key) or False
```

Comment:
```suggestion
        if coordinator.data.get(entity_description.key) is not None
```

pylint does not like the or False